extension-androidauto/README.md

```diff
@@ -28,39 +30,106 @@ allprojects {
     }
 }
 
-// In the app build.gradle file
+// In your build.gradle, add the extension with your other dependencies.
 dependencies {
-  implementation 'com.mapbox.extension:maps-androidauto:10.0.0-rc.9'
+  implementation 'com.mapbox.extension:maps-androidauto:10.4.+'
```

Comment:
would be good to mention that maps sdk dependency should be added separately, since the androidauto extension has compileOnly dependency to the maps SDK.

Reply:
oops, see the mention of adding maps sdk dependency below.